swagger.json

```diff
@@ -126,6 +126,36 @@
             "description": "This resource could be updated at this time by TACO."
           }
         }
+      },
+      "delete": {
+        "summary": "Delete a TACO Resource.",
+        "description": "Deletes a TACO resource (Collection, Digital Repository Object, File metadata object [not binary] or subclass of those).",
```

Comment:
not a binary?

Reply:
This should explicit delete both the File resource (metadata) *and* File binary, if delete resource identifier points to a File. We do not want the case where we delete the File metadata resource but the binary hangs out with no TACO management pointers existing.

Reply:
@aaron-collier can you add this to the description?
> ... deletes both the File resource (metadata) and File binary, if delete resource identifier points to a File

Reply:
Adding now, missed that...